drivers/hmis/app/models/hmis/user.rb

```diff
@@ -125,7 +125,8 @@ def entities_with_permissions(model, *permissions, **kwargs)
   end
 
   private def viewable(model)
-    entities_with_permissions(model, *Hmis::Role.permissions_for_access(:viewable), mode: 'any')
+    # An entity is only considered "viewable" if the user has can_view_project permission for it.
+    entities_with_permissions(model, :can_view_project)
```

Comment:
How does this work with Organizations?  Does it need a special case for that since the relationship to project is inverted?  Or maybe that just lives in entities_with_permission.

Reply:
Ah hm, good question. I'm not sure, I will add a test file for this.

I think you should be able to see the Organization if you have (1) a data source where you have "can_view_project", or (2) an organization where you have "can_view_project", or (3) a project within this org where you have "can_view_project". I _think_ thats how its working now (logic lives in organization viewable_by), but I'll confirm and add tests.

Reply:
Thanks!